app/routes/git_tokens.py

```
@@ -5,192 +5,202 @@
+		res = client.filter(table="git_label", filters={"label": label},
+		                    columns="label, id, git_hosting, masked_token, created_at")
+		return res
+	except Exception as e:
+		raise HTTPException(
+			status_code=status.HTTP_503_SERVICE_UNAVAILABLE,
+			detail="Service temporarily unavailable. Please try again later."
+		) from e
 
 
 @router.post("/", response_model=Dict[str, Any])
 async def add_token(request: Request, payload: AddGitTokenSchema = Body(...)):
```

Ruff (0.11.9)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)